lib/re_captcha/configuration.rb

```diff
@@ -33,7 +36,22 @@ def language_code(locale)
     end
 
     def valid?
-      !private_key.nil? && !public_key.nil?
+      reset_errors
+      set_errors
+      errors.empty?
+    end
```

Comment:
yeah I also  think that pulling active model dep here would be an overkill. While it will reduce the code a bit here. the difference will be just a couple of lines, not a enough significant improvement for me to justify AM inclusion.